src/connection/enterprise/quick_auth_screen.jsx

```diff
@@ -9,7 +9,7 @@ import { authButtonsTheme } from '../../connection/social/index';
 
 // TODO: handle this from CSS
 function icon(strategy) {
-  if (strategy === 'google-apps') return strategy;
+  if (strategy === 'google-apps' || strategy === 'okta') return strategy;
```

Comment:
I am assuming this is for enterprise? The other enterprise connections only return the 'auth0' icon (including PingFederate).

Reply:
Yea, this is for enterprise. I was slightly confused by this section, so hoping  @auth0/dx-sdks-engineer can clarify if this is necessary

Reply:
Just for my understanding - This means the avatar/icon only displays for Okta and Google right? And the other if is for microsoft and everything else displays default auth0 button?

Reply:
That's correct. I'm unaware as to the history as to why it's like that though.